core/src/test/java/org/apache/druid/timeline/VersionedIntervalTimelineTestBase.java

```diff
@@ -98,6 +98,16 @@ static void assertValues(
     Assert.assertEquals(expected, actualSet);
   }
 
+  static void assertSingleElementChunks(
+      PartitionChunk<OvershadowableInteger> expected,
+      PartitionChunk<OvershadowableInteger> actual
+  )
+  {
+    SingleElementPartitionChunk<OvershadowableInteger> expectedSingle = (SingleElementPartitionChunk<OvershadowableInteger>) expected;
```

Comment:
`SingleElementPartitionChunk` is used only by deprecated things such as `NoneShardSpec` and `Tranquility`. We should deprecate `SingleElementPartitionChunk` as well and stop using it.

Reply:
Since it's unrelated to my change, I will leave this one for now as it is. Though I did mark `SingleElementPartitionChunk` deprecated.